### PR TITLE
mrc-3999 Input parameter updates

### PR DESCRIPTION
**Pull request by EmmaLRussell**
This branch implements the changes requested in part 3 of the [MINT update Feb 23 ](https://imperiallondon.sharepoint.com/:w:/r/sites/MRCGIDA-mint-dev/_layouts/15/doc2.aspx?sourcedoc=%7BFFC03FC2-8385-4327-8531-7D69B6DBCA83%7D&file=MINT%20update%20Feb23.docx&action=default&mobileredirect=true&cid=1126be93-7d3e-4d56-87e8-465fe08cdc7f)doc 

Changes:
- Baseline: change "Population Size" label to "Size of population at risk"
-  Interventions: in “Procurement and distribution” allow “When planning procurement, what number of people per net is used?” to increase or decrease in increments of 0.01. 
- Also make all price values step up and down by 0.01
-  In “Price of interventions” set default prices for the three nets to be: pyrethroid-only ITN = $2.0, pyrethroid-PBO ITN = $2.5 and pyrethroid-pyrrole = $3.0 
- Add documentation for price of pyrrole net in 'How to use these settings':
   **Pyrethroid-pyrrole ITN ($USD)**
   Price per Pyrethroid-pyrrole ITN (expected to be replaced every 3-years). 
- Interventions: Change the title of “Intervention coverage potential” section to “Future intervention”. 

Requires this MINT branch which makes required update to vue-dynamic-form: https://github.com/mrc-ide/mint/pull/121
This branch is currently deployed to https://mint-dev.dide.ic.ac.uk

### CODE REVIEW

File: docker/run_app [hunk 1]
```diff
@@ -3,6 +3,6 @@ set -ex
 HERE=$(dirname $0)
 . $HERE/common
 
-export MINT_BRANCH=master
+export MINT_BRANCH=mrc-4049
```
Comment (richfitz) [inline docker/run_app:6]: This can be set back now

File: inst/json/intervention_options.json [hunk 2]
```diff
@@ -93,7 +94,8 @@
                             "label": "Price of pyrethroid LLIN ($USD)",
                             "type": "number",
                             "required": true,
-                            "value": 1.58
+                            "value": 2.00,
```
Comment (richfitz) [inline inst/json/intervention_options.json:97]: Cost of living crisis?

**Review by richfitz — COMMENTED** [review]
LGTM. Note that the step change does not seem to work on previously saved projects! Is that something that you expect?

**Comment by EmmaLRussell** [comment]
> LGTM. Note that the step change does not seem to work on previously saved projects! Is that something that you expect?

Yes, because the project saves the metadata too.

**Comment by EmmaLRussell** [comment]
I've just noticed that the cost effectiveness graph has lost its data! Will try to fix that and write a basic browser test for it...

File: docker/run_app [hunk 3]
```diff
@@ -3,6 +3,6 @@ set -ex
 HERE=$(dirname $0)
 . $HERE/common
 
-export MINT_BRANCH=mrc-4049
+export MINT_BRANCH=master
 
-docker-compose up -d
\ No newline at end of file
+docker-compose up -d --quiet-pull
```
Comment (EmmaLRussell) [inline docker/run_app:8]: I think the problem I was seeing with disappearing graph may have been becaise this wasn't pulling the latest images - I thought docker-compose did that by default. I think this should work?

**Comment by EmmaLRussell** [comment]
I've added browser tests for the cost graphs.

File: docker/run_app (hunk 3 above)
Comment (richfitz) [reply docker/run_app:8]: that makes sense. I think the default is to pull only if a tag is not present, which makes sense with the issues you report

**Review by richfitz — APPROVED** [review]

**Pull request merged**